pkg/pillar/pubsub/pubsub.go

```diff
@@ -7,6 +7,8 @@
 package pubsub
 
 import (
+	"fmt"
+	"github.com/lf-edge/eve/pkg/pillar/pubsub/socketdriver"
```

Comment:
This is not going to work, since `pubsub/socketdriver` imports `pubsub`, and now `pubsub` imports `pubsub/socketdriver`. You are going to end up with an import loop which will fail.